src/Paket.Core/Packaging/PackageMetaData.fs

```diff
@@ -296,30 +296,20 @@ let findDependencies (dependenciesFile : DependenciesFile) config platform (temp
                 |> Seq.map (fun kv -> kv.Value.NugetPackages |> List.map (fun p -> Some kv.Key, p, None))
                 |> List.concat
             | None -> []
-          
+
         [if includeReferencedProjects then
             for proj in project.GetAllReferencedProjects(false,projDeps) |> Seq.filter ((<>) project) do
-                match proj.FindTemplatesFile() with
-                | Some templateFileName when TemplateFile.IsProjectType templateFileName ->
-                    match TemplateFile.Load(templateFileName, lockFile, None, Seq.empty |> Map.ofSeq).Contents with
```

Comment:
The idea of this fix is here: we can't just Load template file for referenced project - we must merge it with info from project (we do so in merge method in PackageProcess.fs).